### PR TITLE
Add Cilium to list of network policy plugins in docs

### PR DESCRIPTION
**Pull request by danwent**
Signed-off-by: Dan Wendlandt <dan@covalent.io>

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4725)

### CODE REVIEW

**Comment by danwent** [comment]
Btw, I have submitted the CLA via email.

**Comment by danwent** [comment]
Thanks for the pointer on the ToC file.   Hopefully this should be better.  

I signed up for the CLA earlier today, including creating an account on the linux foundation site for CNCF CLA.   Maybe its just taking a while for that to propagate through the system?

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+{% capture overview %}
+This page shows how to use Cilium for NetworkPolicy.
+
+For background on Cilium, read the [Introduction to Cilium](http://cilium.readthedocs.io/en/latest/intro/).
+{% endcapture %}
+
+{% capture prerequisites %}
+* Install Cilium for Kubernetes.
```
Comment (chenopis) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: Perhaps the prerequisite should simply be to have a Kubernetes cluster up and running. You can replace this line with

`* {% include task-tutorial-prereqs.md %}`

That will include the generic requirements.

**Comment by chenopis** [comment]
@danwent You might need to check to make sure that the email address in the commit matches the one used to sign the CLA. These links might be helpful:
https://help.github.com/articles/setting-your-email-in-git/
http://stackoverflow.com/questions/2919878/git-rewrite-previous-commit-usernames-and-emails

File: docs/tasks/administer-cluster/cilium-network-policy.md (hunk 1 above)
Comment (danwent) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: @chenopis  Thanks!  Just pushed a new version with this.

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+daemonset "cilium" created
+```
+
+The remainder of the Getting Started Guide explains how to enforce both L3/L4 (i.e., IP address + port) security
+policies, as well as L7 (e.g., HTTP) security policies using an example application.
+
+## Deploying Cilium for Production Use
+
+For detailed instructions around deploying Cilium for production, see:
+[Cilium Administator Guide](http://cilium.readthedocs.io/en/latest/admin/) This
```
Comment (bowei) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: spelling administrator

**Comment by danwent** [comment]
Thanks.  I've doubled checked and as far as I can tell, the commit uses the same <my-first-name>@covalent.io that I used to register the CLA.   Any ideas on what I can do to further debug?

File: docs/tasks/administer-cluster/cilium-network-policy.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+```console
+NAME            DESIRED   CURRENT   READY     NODE-SELECTOR   AGE
+cilium          1         1         1         <none>          2m
+...
+```
+
+There are two main components to be aware of:
+
+- One `cilium` Pod runs on each node in your cluster and enforces network policy on the traffic to/from Pods on that node using Linux BPF.
+- For production deployments, Cilium should leverage the kev-value store cluster (e.g., etcd) used by Kubernetes, which typically runs on the Kubernetes master nodes.   The [Cilium Administator Guide](http://cilium.readthedocs.io/en/latest/admin/) includes an example DaemonSet which can be customized to point to this key-value store cluster.   The simple ''all-in-one'' DaemonSet for minikube requires no such configuration because it automatically deploys a `cilium-consul` Pod to provide a key-value store.
```
Comment (bowei) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: spell kev -> key

**Comment by chenopis** [comment]
@danwent You could also try connecting your GitHub account w/ a Linux Foundation account. Just go to https://identity.linuxfoundation.org/?destination=node/285/individual-signup and use "Log in with GitHub" button.

screen shot 2017-08-11 at 12 25 48 am

**Comment by chenopis** [comment]
@danwent Also, sometimes people say that the email w/ the link to the CLA to e-sign takes a while to arrive or ends up in their spam folder.

**Comment by danwent** [comment]
Thanks @chenopis .   I actually already logged in with github to that page so I think they are already linked.

I checked my spam folder and did not see anything I missed. 

At this point I just sent an email to cla@cncf.io to see if there is any aspect of the process they do not consider complete, or if there is someone there who can help troubleshoot the issue.   Thanks!

**Comment by danwent** [comment]
screen shot 2017-08-15 at 9 30 15 am

File: docs/tasks/administer-cluster/cilium-network-policy.md (hunk 3 above)
Comment (danwent) [inline docs/tasks/administer-cluster/cilium-network-policy.md]: @bowei, whoops, thanks for pointing out the typos.  I fixed these and ran ispell as well to make sure I hadn't missed anything else.

**Comment by danwent** [comment]
Hi, just trying to understand if this is waiting on me to update my branch again, or not.  Thanks!

**Comment by chenopis** [comment]
@danwent Nope. We were inundated with several waves of PRs, so I'm just now catching up. :( Thanks for the contribution! =)

**Comment by danwent** [comment]
Woohoo, thanks!  

No worries on the delay. I just wasn't sure on the process myself.   Thank again!

**Comment by danwent** [comment]
Unfortunately, I'm seeing some breakage when I click on the Cilium link on this page: https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy/  .   

It seems like all links on that page (Cilium included) link to a filename in https://kubernetes.io/docs/tasks/configure-pod-container/  .   When I click on the ciliu link (https://kubernetes.io/docs/tasks/configure-pod-container/cilium-network-policy/) I get a 404.   When I click on the calico link (https://kubernetes.io/docs/tasks/configure-pod-container/calico-network-policy/) it somehow gets redirected to https://kubernetes.io/docs/tasks/administer-cluster/calico-network-policy/ , which works.   A similar link with administer-cluster instead of configure-pod-container for Cilium (https://kubernetes.io/docs/tasks/administer-cluster/cilium-network-policy/) also works if I access it directly.   So it seems like there is some logic that rewrites the URL from configure-pod-container -> administer-cluster that is working for all other pages, but not for cilium.   

Should I have just made the link administer-cluster from the start?   I just was just copy-pasting from the previous plugin links, which seems to have gotten me in trouble.  

Thanks!

**Comment by chenopis** [comment]
@danwent Thanks for fixing that link!

**Pull request merged**